### DISCUSSION TITLE
new helmfile/helmfile missing 2 tags from  roboll/helmfile/tags 302 ( v0.144.0 0.143.5 )

### DISCUSSION THREAD

**Discussion by dalain007** [helmfile/helmfile#63] [Questions]
hello

trying to figure out if we should start using **helmfile/helmfile** repo or keep on using release form **roboll/helmfile**
look like there is a mismatch between the tags in both repo

as of today Apr 25th, 2022, we can see 

https://github.com/roboll/helmfile/tags  has **302** tags 

- https://github.com/roboll/helmfile/releases/tag/v0.144.0  **26 days ago**
- https://github.com/roboll/helmfile/releases/tag/v0.143.5  **28 days ago**
- https://github.com/roboll/helmfile/releases/tag/v0.143.4  **Mar 24, 2022**

https://github.com/helmfile/helmfile/tags has **300** tags
- https://github.com/helmfile/helmfile/releases/tag/v0.143.4  **Mar 24, 2022**

**Comment by yxxhero** [comment]
@dalain007 you can use helmfile/helmfile directly.
